include/osquery/filesystem.h

```diff
@@ -351,6 +351,33 @@ Status procReadDescriptor(const std::string& process,
  * @return status The status of the read.
  */
 Status readRawMem(size_t base, size_t length, void** buffer);
-
 #endif
+
+/*
+ * @brief A function to archive files specified into a single file
+ *
+ * Given a set of paths we bundle these into a tar archive.
```

Comment:
Would you mind adding the `@param` specifications as you have below? I know this wasn't something you wrote *blushes* >.>